concent_api/core/views.py

```diff
@@ -0,0 +1,223 @@
+import json
+import datetime
+
+from django.http                    import HttpResponse
+from django.views.decorators.http   import require_POST
+from django.utils                   import timezone
+from django.conf                    import settings
+
+from utils.api_view                 import api_view, Http400
+from .models                        import Message, MessageStatus
+
+
+@api_view
+@require_POST
+def send(request, message):
+    data = message
+    validate_message(data)
+
+    if data['type'] == "MessageForceReportComputedTask":
```

Comment:
IMHO wszystkie stringi powinny być gdzieś zdefiniowane, w tym wypadku skoro dotyczą modelu Message, można dodać tam klasę imitującą enum według tej konwencji:
http://blog.richard.do/index.php/2014/02/how-to-use-enums-for-django-field-choices/

Do tego dodać metody typu:
```
def is_type_message_force_report_computed_task():
    return self.type == TYPES.MessageForceReportComputedTask
```

I wtedy zamiast powtarzać stringi w róznych miejscach użyć tych metod.

Reply:
It's actually how we're going to do this. It's already rewritten in @kbeker's branch to use constants from `golem-messages`, like this: `MessageForceReportComputedTask.TYPE`.

Also after the transition to `golem-mesages` we can do `isinstance(message, MessageForceReportComputedTask)` instead.

The only reason why a bare string is used here is that it's the transitional JSON-based version that was going to be completely rewritten from the beginning. It was based on my initial, very loose specs and I wanted @Jakub89 to implement it quickly, while the final shape of the messages we require wasn't even decided upon yet.

Reply:
:+1: